integration-tests/chopsticks/src/command/network.ts

```diff
@@ -0,0 +1,28 @@
+import { mainChains } from '../network/index.js'
+import { setStorage, setupNetwork } from '../network/utils.js'
+
+export async function createTestNetwork() {
+	const relayConfig = mainChains.polkadot.getConfig({})
+
+	// 1. Create the network specific state
+	const kiltStorage = mainChains.kilt.storage.assignNativeTokensToAccounts([
+		'4pF5Y2Eo6doQHPLQj5AkndZwtomVB8ab2sRftRS2D9JDdELr',
+	])
+
+	// 2. Create the network specific config
+	const kiltConfig = mainChains.kilt.getConfig({})
+
+	// 2. Add the different parachainContexts here. The order of parachainContexts and parachainStorage should match.
+	const parachainOptions = [kiltConfig]
+	const parachainStorage = [kiltStorage]
+
+	const { relayChainContext, parachainContexts } = await setupNetwork(relayConfig, parachainOptions)
+
+	parachainStorage.map(async (storage, index) => await setStorage(parachainContexts[index], storage))
+
+	await Promise.all(
+		[...parachainContexts, relayChainContext].map((context) => {
+			context.pause()
+		})
```

Comment:
```suggestion
		[...parachainContexts, relayChainContext].map((context) => context.pause())
```
Nothing to await if there's nothing returned; either remove the curly brackets or add an explicit return before `context.pause()`